libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -904,6 +904,10 @@ void IRGeneratorForStatements::endVisit(MemberAccess const& _memberAccess)
 		{
 			solUnimplementedAssert(false, "");
 		}
+		else if (member == "interfaceId")
+		{
+			solUnimplementedAssert(false, "");
```

Comment:
Should be almost the same code as in ExpressionCompiler.cpp.

Reply:
Oh nice, the code is really almost identical. First time that I touched Yul generation :)